### PR TITLE
Fix redirect after login for proxy mode, issue#1182

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@adrianschroeter Please have a look. I tested this on my machine, but still have to test this in proxy mode on our dev system

**Comment by adrianschroeter** [comment]
I am fine with merging it and testing it on build-test.opensuse.org.
Can you add the [webui] tag?

**Comment by bgeuken** [comment]
@adrianschroeter done

File: src/api/app/controllers/webui/user_controller.rb [hunk 1]
```diff
@@ -42,7 +42,6 @@ def do_login
     end
 
     if user.nil? || (user.state == User::STATES['ichainrequest'] || user.state == User::STATES['unconfirmed'])
```
Comment (bgeuken) [inline src/api/app/controllers/webui/user_controller.rb]: @adrianschroeter btw it looks to me like the state check has no effect. Or am I missing something?

Comment (bgeuken) [inline src/api/app/controllers/webui/user_controller.rb]: Ah, wait. So the ichain request go through a different controller?

**Pull request merged**